homeassistant/components/qbus/entity.py

```diff
@@ -61,7 +64,7 @@ class QbusEntity(Entity, ABC):
     _attr_name = None
     _attr_should_poll = False
 
-    def __init__(self, mqtt_output: QbusMqttOutput) -> None:
+    def __init__(self, hass: HomeAssistant, mqtt_output: QbusMqttOutput) -> None:
```

Comment:
this is not used

Reply:
Oh right, otherwise mypy will complain.
You could move the initialization of the Debouncer to `async_added_to_hass`, since there `self.hass` is initialized already, and keep only the type definition of the debouncer on `__init__` (like `flux_led/number.py`) does,

That way we avoid having to pass `hass` around to all platforms and to the base init